src/styles/pagination.scss

```diff
@@ -7,94 +7,94 @@
 
 $block: #{$fd-namespace}-pagination;
 
-.#{$block} {
+@mixin fd-pagination-mobile() {
+  .#{$block}__per-page + .#{$block}__nav {
+    @include fd-set-margins-x('auto', 0);
+  }
+
+  .#{$block}__total,
+  .#{$block}__per-page-label,
+  .#{$block}__link,
+  .#{$block}__more {
+    display: none;
+  }
 
-  $fd-pagination-link-padding-x: 0.25rem;
-  $fd-pagination-padding-x: 0.5rem;
+  .#{$block}__button--mobile,
```

Comment:
I haven't found `fd-pagination__button` class that you are modifying with `fd-pagination__button--mobile`. You can't have the second without the first :)

Reply:
<img width="936" alt="Screenshot 2021-11-22 at 12 12 52" src="https://user-images.githubusercontent.com/20265336/142843272-c0b477c8-bfa2-4344-b1c7-25bbe4f58972.png">